tests/system/test_docker_marathon_on_marathon.py

```diff
@@ -0,0 +1,66 @@
+""" Test using marathon on marathon (MoM).
+    This test suite imports all common tests found in marathon_common.py which are
+    to be tested on root marathon and MoM.
+    In addition it contains tests which are specific to MoM environments only.
+"""
+
+import apps
+import common
+import os
+import pytest
+import retrying
+import scripts
+import shakedown
+import time
+
+from datetime import timedelta
+from dcos import mesos
+from shakedown import marathon
+
+# the following lines essentially do:
+#     from marathon_common_tests import test_*
+import marathon_common_tests
+for attribute in dir(marathon_common_tests):
```

Comment:
Hm, I'm not sure this is the right approach. Are there other files that should not be imported? Why not import them explicitly by name?